### DISCUSSION TITLE
Change macOS "About example-app" Popup

### DISCUSSION THREAD

**Discussion by nshardy** [tauri-apps/tauri#7945] [Q&A]
I hope I worded the title correctly, I'm hoping this is something that is doable. I was hoping to be able to change the information inside this menu:

<img width="355" alt="Screenshot 2023-10-02 at 7 36 05 PM" src="https://github.com/tauri-apps/tauri/assets/104463569/c9f42dcf-9b3a-4394-94e4-7594d344df25">
<img width="396" alt="Screenshot 2023-10-02 at 7 40 50 PM" src="https://github.com/tauri-apps/tauri/assets/104463569/3055645b-1247-4668-a3d0-04293cd0573c">

I'm not expecting a reply any time soon, but I feel like there could possibly be a way to change that information inside of the 'src-tauri/src/main.rs' file, or the 'src-tauri/tauri.conf.json' file.
I have done multiple Google searches and StackOverflow searches, but I haven't found anything.
<br>
I hope this is possible, but if it isn't this could be a feature worth adding, especially for macOS. I know I'm not the only one that wants to have Third-Party, or self-made, applications that feel native to macOS.

<img width="396" alt="Screenshot 2023-10-02 at 7 45 40 PM" src="https://github.com/tauri-apps/tauri/assets/104463569/148b84ed-f6be-40d0-9c52-71dea1dbb1f9">

**Comment by mimi1405** [comment]
In Tauri Conf Json file you can do the necessary changes to do that. Watch out for the version. You might add there in the same string more then the version to let it display.

**Reply by nshardy** [reply]
That would be a good workaround. Thank you!

**Reply by nshardy** [reply]
> In Tauri Conf Json file you can do the necessary changes to do that. Watch out for the version. You might add there in the same string more then the version to let it display.

This approach doesn't work as it states that only the version is allowed to be in the version string. I tried this a while back and meant to reply since.

**Comment by Blatman** [comment]
Been a while since I built a V1 Tauri app but I'm certain the info is taken from the Info.plist file in time-honoured Apple fashion.
In particular -
CFBundleShortVersionString
CFBundleVersion
NSHumanReadableCopyright

Currently struggling with the V2 beta but I am assuming the about menu still uses the normal Apple hooks at the low level.


<img width="280" alt="about" src="https://github.com/tauri-apps/tauri/assets/11089335/694380a6-f7d8-4cd6-ae17-4404c82ef5ab">

<img width="406" alt="about2" src="https://github.com/tauri-apps/tauri/assets/11089335/95351c34-2520-49db-97cf-7fc51282e870">
